src/auth/screens/language-setting.screen.js

```diff
@@ -7,9 +7,11 @@ import { ListItem } from 'react-native-elements';
 import { colors, fonts } from 'config';
 import { changeLocale } from 'auth';
```

Comment:
Currently, changing the language in the language settings screen doesn't update the language so we'll just need to map it to our new library.